### PR TITLE
Blender: Let texture export use numeric constants.

### PR DESCRIPTION
**Pull request by tschw**

### CODE REVIEW

**Comment by repsac** [comment]
edits look pretty straight forward, looks fine

File: utils/exporters/blender/addons/io_three/constants.py [hunk 1]
```diff
@@ -32,6 +32,26 @@
     'SPHERICAL_REFLECTION': 'SphericalReflectionMapping'
 })
 
+NUMERIC = {
```
Comment (mrdoob) [inline utils/exporters/blender/addons/io_three/constants.py]: How about doing something like this instead?

```
 MAPPING_TYPES = type('Mapping', (), {
     'UV': 300, #UVMapping
     'CUBE_REFLECTION': 301, #CubeReflectionMapping
     'CUBE_REFRACTION': 302, #CubeRefractionMapping
     'SPHERICAL_REFLECTION': 305 #SphericalReflectionMapping
 })
```

Comment (tschw) [inline utils/exporters/blender/addons/io_three/constants.py]: Won't work - `Mapping` is for strings and used for the UI.

Comment (mrdoob) [inline utils/exporters/blender/addons/io_three/constants.py]: Ah, I see.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**